src/game_initialization/lobby_info.hpp

```diff
@@ -65,7 +65,7 @@ class lobby_info
 
 	/** Adds a new filter function to be considered when @ref apply_game_filter is called. */
 	void add_game_filter(game_filter_func func)
-	{
+	{	
```

Comment:
Please remove this extraneous trailing whitespace.

Reply:
Sure. I'll adjust these when I add the preferences.